frontend/src/metabase/components/AccordianList.jsx

```diff
@@ -56,20 +60,50 @@ export default class AccordianList extends Component {
 
     static defaultProps = {
         style: {},
+        width: 300,
         searchable: (section) => section.items && section.items.length > 10,
         alwaysTogglable: false,
         alwaysExpanded: false,
         hideSingleSectionTitle: false,
     };
 
+
+    // TODO: reimplement scrolling currently selected item into view
+
     componentDidMount() {
-        // when the component is mounted and an item is selected then scroll to it
-        const element = this.refs.selected && ReactDOM.findDOMNode(this.refs.selected);
-        if (element && element.scrollIntoView && !elementIsInView(element)) {
-            element.scrollIntoView();
+        this._forceUpdateList();
```

Comment:
Would be nice to have a note here about why this is needed.

Reply:
Done.